server/detectVersion.ts

```diff
@@ -0,0 +1,9 @@
+const rCapiVersion = /^https:\/\/(?:capi.dev\/|deno.land\/x\/capi)@(.+?)\//
+
+export function detectVersion(): string {
+  const version = rCapiVersion.exec(import.meta.url)?.[0]
+  if (!version) {
+    throw new Error("Could not detect version from URL. Please specify via `--version` flag.")
```

Comment:
if the catch block in `version` doesn't make use of these errors arent the error messages here moot?

Reply:
No, we want this error message to occur in the case of running the sync command of a non-dist Capi without `--version`